vm/rust/src/jsonrpc.rs

```diff
@@ -50,7 +52,7 @@ pub struct TransactionTrace {
     #[serde(skip_serializing_if = "Option::is_none")]
     constructor_invocation: Option<FunctionInvocation>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    function_invocation: Option<FunctionInvocation>,
+    function_invocation: Option<ExecuteInvocation>,
```

Comment:
`ExecuteInvocation = Result<FunctionInvocation>`, I think the name is a bit confusing and maybe we can change it in the future. Just making a note of this here. Maybe `FunctionInvocationResult` or another one more original but still relatable

Reply:
Issue at https://github.com/NethermindEth/juno/issues/2991